Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -83,12 +90,21 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
     override func viewDidLoad() {
         view.backgroundColor = UIColor.white
 
-        // scale the slides down for iPhone 4S
+        // scale the slides down for small devices
         if view.frame.height <=  480 {
-            slideVerticalScaleFactor = 1.33
+            slideVerticalScaleFactor = 1.33 // 4S
+        } else if view.frame.height <= 568 {
+            slideVerticalScaleFactor = 1.15 // SE
+        }
+        
+        let cardSlides: [String]
+        if delegate?.introViewControllerShouldAskForNotifications(self) == true {
```

Comment:
Nit: rewrite conditional without `== true`.

You're using this 3 times in this method, so will be worth making a local variable out of it.